python/lsst/ip/isr/isrTask.py

```diff
@@ -2276,7 +2293,7 @@ def maskAndInterpolateDefects(self, exposure, defectBaseList):
         """
         self.maskDefect(exposure, defectBaseList)
         self.maskEdges(exposure, numEdgePixels=self.config.numEdgeSuspect,
-                       maskPlane="SUSPECT")
+                       maskPlane="SUSPECT", level=self.config.edgeMaskLevel)
         isrFunctions.interpolateFromMask(
             maskedImage=exposure.getMaskedImage(),
             fwhm=self.config.fwhm,
```

Comment:
I have problems with this whole function, but not your changes to it. But while looking at it I can't help but wonder what part of `maskAndInterpolateDefects()` implies that edges will also be masked. Neither the function name, nor the docstring, mention that this will happen.

Reply:
I don't believe this function is called directly anymore.  `ip_isr` probably needs a cleanup ticket to get rid of outdated code.